tests/unit-core/test-container.c

```diff
@@ -122,6 +167,73 @@ main (void)
   jerry_gc (JERRY_GC_PRESSURE_LOW);
   TEST_ASSERT (global_counter == 1);
 
+  run_eval ("function assert (v) {\n"
+            "  if (v !== true)\n"
+            "    throw 'Assertion failed!'\n"
+            "}");
+
+  run_eval ("function test_values (arr1, arr2) {\n"
+            "  assert (arr1.length == arr2.length);\n"
+            "  for (let i = 0; i < arr1.length; i++) {\n"
+            "    assert (arr1[i] == arr2[i]);\n"
+            "  }\n"
+            "}\n");
+
+  run_eval ("var entries_check_array = [1, 3.14, 2, true, 3, 'foo'];\n"
+            "var values_check_array = [3.14, true, 'foo'];\n"
+            "var keys_check_array = [1, 2, 3,];\n"
+            "var obj = { x:3, y:'foo'};\n"
+            "var b_int = 1n;\n"
+            "var object_check_array = [1, obj];\n");
+
+  run_eval ("var map = new Map();\n"
+            "map.set(1,3.14);\n"
+            "map.set(2,true);\n"
+            "map.set(3,'foo');\n"
+            "var set = new Set();\n"
+            "set.add(3.14);\n"
+            "set.add(true);\n"
+            "set.add('foo');\n"
+            "var obj_map = new Map();\n"
+            "obj_map.set(1, obj);\n");
+
+  run_eval ("var result = create_array_from_container (map, true);\n"
```

Comment:
You are put a space before `(` but not in the above cases. Please be consistent.